python/ray/tests/test_structured_logging.py

```diff
@@ -122,6 +122,47 @@ def test_record_with_user_provided_context(self, shutdown_only):
         assert "exc_text" not in record_dict
 
 
+class TestLogfmtFormatter:
+    def test_empty_record(self, shutdown_only):
+        formatter = LogfmtFormatter()
+        record = logging.makeLogRecord({})
+        formatted = formatter.format(record)
```

Comment:
@jjyao Should we maintain a function to parse a logfmt string into a dictionary, or is it enough to check whether the key is in the string?